media-libs/sdl2-pango/sdl2-pango-2.1.5.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="SDL2 port of SDL_Pango for rendering text using Pango"
+HOMEPAGE="https://github.com/markuskimius/SDL2_Pango/"
+SRC_URI="https://github.com/markuskimius/SDL2_Pango/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE="static-libs"
```

Comment:
Is this needed? Nothing seems to depend on this library with static-lib requirement, so it should just be disabled.
https://projects.gentoo.org/qa/policy-guide/installed-files.html#pg0302

Also:
https://projects.gentoo.org/qa/policy-guide/installed-files.html#pg0303
but I see you're handling `src_install` properly already.